pynamodb/connection/base.pyi

```diff
@@ -58,6 +58,7 @@ class Connection:
     def get_item_collection_map(self, return_item_collection_metrics): ...
     def get_exclusive_start_key_map(self, table_name: Text, exclusive_start_key): ...
 
+    def get_operation_kwargs_for_delete_item(self, *args, **kwargs) -> Dict: ...
```

Comment:
Can you add all of the `get_operation_kwargs..` functions here?

Reply:
ye